### PR TITLE
Add SIG Release teams

### PR DESCRIPTION
**Pull request by justaugustus**
Fixes: kubernetes/sig-release#241

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/sig release

### CODE REVIEW

**Comment by justaugustus** [comment]
(this is not ready to go yet, I just want to see if we'll run into any failures with comments next to the usernames.)

**Comment by cblecker** [comment]
/hold
LMK when this is ready :)

**Comment by justaugustus** [comment]
this is good to go now!
/hold cancel

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    description: Team members can use `/milestone` or `/status` commands on issues/PRs
+    maintainers:
+    - calebamiles # Release / PM
+    - justaugustus # Azure / PM / Release
+    - tpepper # Release
+    - spiffxp # 1.14 RT Lead / Testing
+    members:
+    - abgworrall # GCP
+    - andrewsykim # Cloud Provider
+    - apsinha # PM
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    - bgrant0607 # Architecture
+    - bradamant3 # Docs
+    - brancz # Instrumentation
+    - bsalamat # Scheduling
+    - cantbewong # VMware
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    - enj # Auth
+    - feiskyer # Azure
+    - fejta # Testing
+    - floreks # UI
+    - foxish # Big Data
+    - frapposelli # VMware
+    - hogepodge # Cloud Provider / OpenStack
+    - idvoretskyi # PM
+    - jagosan # Cloud Provider
+    - jboyd01 # Service Catalog
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    - jdumars # Architecture
+    - justinsb # AWS
+    - k82cn # Scheduling
+    - khahmed # IBM Cloud
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    - khenidak # Azure
+    - kibbles-n-bytes # Service Catalog
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    - mikedanese # Auth
+    - mwielgus # Autoscaling
+    - nikhita # ContribEx
+    - parispittman # ContribEx
+    - patricklang # Windows
+    - phillels # ContribEx
+    - piosz # Instrumentation
+    - prydonius # Apps
+    - pwittrock # CLI
+    - rjmorse # OpenStack
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+    - quinton-hoole-2 # Multicluster
+    - roberthbailey # Cluster Lifecycle
+    - rtheis # IBM Cloud
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Not an org member

File: config/kubernetes/sig-release/teams.yaml [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+    - saad-al # Storage
```
Comment (cblecker) [inline config/kubernetes/sig-release/teams.yaml]: Typo

**Comment by justaugustus** [comment]
Should we concerned by the fact that people listed as SIG Chairs/TLs aren't also org members?
I've removed the ones you mentioned and fixed Saad's username, but we should take a look at that in the future.

File: config/kubernetes/sig-release/teams.yaml [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+    - jagosan # Cloud Provider
+    - jdumars # Architecture
+    - justinsb # AWS
+    - k82cn # Scheduling
+    - khenidak # Azure
+    - kow3ns # Apps
+    - kris-nova # AWS
+    - lavalamp # API Machinery
+    - liyinan926 # Big Data
+    - luxas # Cluster Provider
```
Comment (nikhita) [inline config/kubernetes/sig-release/teams.yaml]: typo: s/Cluster Provider/Cluster Lifecycle :P

Comment (justaugustus) [reply config/kubernetes/sig-release/teams.yaml]: Thanks for catching that, @nikhita!

**Comment by cblecker** [comment]
@justaugustus Yes, it's probably something we should look into. I've poked sig leads before to go through the membership process.
/shrug

/lgtm
/approve

**Comment by cblecker** [comment]
I have manually synced this, so you should be good to go.

**Pull request merged**